### PR TITLE
qa_crowbarsetup.sh: Deploy Manila for cloud 6 and newer

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by aplanas** [comment]
+1

**Comment by toabctl** [comment]
I think there is currently no manila proposal.

**Comment by bmwiedemann** [comment]
:-1: test failed
```
crowbar manila proposal create default
Usage: crowbar <area> <subcommand>
```

**Comment by dirkmueller** [comment]
@bmwiedemann code has been fixed

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -2218,6 +2218,9 @@ function deploy_single_proposal()
         ceph)
             [[ -n "$deployceph" ]] || continue
             ;;
+        manila)
+            iscloudver 6plus || continue
```
Comment (toabctl) [inline scripts/qa_crowbarsetup.sh]: Shouldn't that be "5plus" ?

**Comment by tboerger** [comment]
Tests fail but otherwise LGTM

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (toabctl) [inline scripts/qa_crowbarsetup.sh]: > Shouldn't that be "5plus" ?

Hm. No. Cloud6 should be fine for now.

**Comment by MaximilianMeister** [comment]
and does it make sense to apply it by default? especially if it is not fully integrated with HA/SSL and so on.

**Comment by vuntz** [comment]
@MaximilianMeister why not? We do it for trove, which has no HA/SSL, etc. What matters is that we want to make sure that it works :-)

**Comment by toabctl** [comment]
+1 . Tested manually with Cloud 6 and works for me. Should be merged after milestone 1.

**Comment by rsalevsky** [comment]
@toabctl Should we also provide the possibility to deploy manila on cloud5?

**Comment by toabctl** [comment]
> @toabctl Should we also provide the possibility to deploy manila on cloud5?

Yes. But let's start with Cloud 6 first.

**Comment by rsalevsky** [comment]
Ok +1

**Comment by MaximilianMeister** [comment]
probably not directly related to this code here, but it still failed at manila proposal
```
crowbar manila proposal commit default
/opt/dell/bin/barclamp_lib.rb:536:in `eval': Net::ReadTimeout (Net::ReadTimeout)
...
```

**Comment by toabctl** [comment]
@dirkmueller can you rebase that commit to give the CI system a chance to succeed?

**Comment by bmwiedemann** [comment]
needs rebase to get the curl -L fix to pass the test

**Comment by dirkmueller** [comment]
pull #433 is going to help this test to succeed

**Comment by toabctl** [comment]
Succeeded with `develcloud5`. Retest with `develcloud6`.

**Comment by toabctl** [comment]
@dirkmueller I guess this needs another rebase...

**Pull request merged**